spyder/plugins/switcher/plugin.py

```diff
@@ -89,3 +146,85 @@ def on_main_menu_teardown(self):
 
     # --- Public API
     # ------------------------------------------------------------------------
+
+    # Switcher methods
+    def set_placeholder_text(self, text):
+        """Set the text appearing on the empty line edit."""
+        self._switcher.set_placeholder_text(text)
+
+    def setup(self):
+        """Set-up list widget content based on the filtering."""
+        self._switcher.setup()
+
+    def set_position(self, top, parent=None):
+        """Set the position of the dialog."""
+        self._switcher.set_position(top, parent)
+
+    # QDialog methods
+    def show(self):
+        """Show switcher widget with QT hide method for QDialog."""
+        self._switcher.show()
+
+    def hide(self):
+        """Hide switcher widget with QT hide method for QDialog."""
+        self._switcher.hide()
+
+    def close(self):
+        """Close switcher widget with QT close method for QDialog."""
+        self._switcher.close()
+
+    def setVisible(self, visible):
+        """Show or hide switcher widget with QT hide method for QDialog."""
```

Comment:
```suggestion
    def set_visible(self, visible):
        """Show or hide switcher."""
```

We don't use camel-cased names for our own methods. And I think this requires a small change in `EditorSwitcherManager` for it to work.

Reply:
Oops you are right! sorry.